README.md

```diff
@@ -82,8 +82,8 @@ Version Control
 
 <p align="center">
 <a>
-  <img height="180em" src="https://github-readme-stats.vercel.app/api?username=Arsenic-ATG&show_icons=true&hide_border=true&count_private=true&theme=nord"/>
-  <img height="180em" src="https://github-readme-stats.vercel.app/api/top-langs/?username=Arsenic-ATG&hide=TeX,QMake&theme=nord&layout=compact&hide_border=true"/>
+  <img height="180em" src="https://grs-delta-taupe-49.vercel.app/api?username=Arsenic-ATG&show_icons=true&hide_border=true&count_private=true&theme=nord"/>
```

Comment:
the actual deployed project is private clone of anurag hazra's project ( I didn't went for a fork as I once accidentally created a pull requests there ) and most probably would be updated manually when a new feature comes out.